core/src/main/java/org/lint/azzert/visitor/LintAssertAnnotationVisitor.java

```diff
@@ -0,0 +1,25 @@
+package org.lint.azzert.visitor;
+
+import org.lint.azzert.context.Context;
+import org.objectweb.asm.AnnotationVisitor;
+
+public class LintAssertAnnotationVisitor extends AnnotationVisitor {
```

Comment:
See, in JUnit, methods marked with @Ignore and @Disabled are not considered to be tests unless they are also annotated with @Test. That's why I chose not to rely on the AnnotationVisitor and use the ExemptDisabledTestsCommand to filter out all @Test methods with @Ignore/@Disabled annotation

Reply:
TestNG does not have @ignore but used @Test(enabled=false) , need to use AnnotationVisitor to process the parameters. This approach could also be used to filter out the false negatives of @Test(expected=SomeException.class) in JUnit 4